.github/workflows/cd.yml

```diff
@@ -29,7 +34,30 @@ jobs:
           field: aiKey
           value: ${{ secrets.PUBLIC_AIKEY }}
 
+      - name: update visx for preview
+        if: ${{ github.event.inputs.preview == 'true'}}
+        run: |
+          VERSION=$(jq -r .version ./package.json)
+          echo '----------------- current version:' $VERSION
+          MINOR_VER=$(echo $VERSION | awk -F. '{print $2}')
+          DATE_WITH_TIME=`date "+%Y%m%d%H"`
+          if [ $((MINOR_VER%2)) -eq 0 ]; then
+            VERSION=$(echo ${VERSION%-*} | awk -v val=$DATE_WITH_TIME -F. '/[0-9]+\./{$2++;$3=val;print}' OFS=.)
+          else
+            VERSION=$(echo ${VERSION%-*} | awk -v val=$DATE_WITH_TIME -F. '/[0-9]+\./{$3=val;print}' OFS=.)
+          fi
+          echo '----------------- prerelease version:' $VERSION
+          jq --arg VERSION "$VERSION" '.version=$VERSION' package.json > tmp.$$.json
+          mv tmp.$$.json package.json
+
+      - name: Build visx for preview
+        if: ${{ github.event.inputs.preview == 'true'}}
```

Comment:
is else supported? If yes would be more cleaner to have just one build vsix

Reply:
done